packages/server/modules/automations/migrations/20230905162038_automations.ts

```diff
@@ -0,0 +1,51 @@
+import { Knex } from 'knex'
+
+const AUTOMATIONS_TABLE_NAME = 'automations'
+const AUTOMATION_RUNS_TABLE_NAME = 'automation_runs'
+
+const AUTOMATION_ID = 'automationId'
+const AUTOMATION_REVISION_ID = 'automationRevisionId'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable(AUTOMATIONS_TABLE_NAME, (table) => {
+    table.string(AUTOMATION_ID).notNullable
+    table.string(AUTOMATION_REVISION_ID).notNullable()
+    table.string('automationName').notNullable()
+    table.string('projectId').notNullable().references('id').inTable('streams')
+    table.string('modelId').notNullable()
```

Comment:
why not use a FK here?

Reply:
because currently the `modelId` is actually the `branchName`.

We need to migrate our webhooks to send the `modelId` in the payload, plus ( a bigger issue than the previous ) our SDK-s need to support using it.

So for now it cannot be an FK